### PR TITLE
adding company and source

### PR DESCRIPTION
**Pull request by viskvortsov**
- added company
- added source

### CODE REVIEW

**Comment by viskvortsov** [comment]
@pfrisella
Would you like us to do something else?

File: sources/vashkpi.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "kpi_indicators",
```
Comment (pfrisella) [inline sources/vashkpi.json:2]: Review the instructions at: https://github.com/googledatastudio/ds-data-registry#add-a-source

ID should match filename, all uppercase.

File: sources/vashkpi.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "name": "KPI: Indicators",
+  "categories": ["DATA_COLLECTION"],
+  "organization": "vashkpi",
```
Comment (pfrisella) [inline sources/vashkpi.json:5]: uppercase

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/vashkpi.json (hunk 1 above)
Comment (viskvortsov) [reply sources/vashkpi.json]: Done

**Comment by viskvortsov** [comment]
@pfrisella thanks for your remarks.
I've commited new version

**Review by pfrisella — APPROVED** [review]

**Pull request merged**